app/jobs/regular/discourse_reactions/post_action_synchronizer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Jobs
+  module DiscourseReactions
+    class PostActionSynchronizer < ::Jobs::Base
+      def execute(args = {})
+        ::DiscourseReactions::ReactionPostActionSynchronizer.sync!
```

Comment:
what triggers this, should this not be a "every 1 day" or something like that?

Reply:
oh I see, setting change... so initial sync will never fire... we need to make this a regular job instead and check if it "needs to synchronize"

Reply:
No it's not necessary, this only is triggered when the `discourse_reactions_excluded_from_like` setting changes for backfilling:

https://github.com/discourse/discourse-reactions/blob/e6f3ca690e6e9d67bfe6350421424fe5f5c27cb7/plugin.rb#L406-L411

The `ReactionManager` is used for general usage when you are clicking in the UI. Internally, we will set `discourse_reactions_like_sync_enabled` to `true` then manually trigger this job for testing.